lib/ansible/modules/cloud/amazon/ecs_taskdefinition.py

```diff
@@ -97,6 +97,11 @@
         options:
           awslogs-group: ecs
           awslogs-region: us-west-2
+      environment:
+         -  name: "FAASOS_ENV"
```

Comment:
You don't need to quote those strings. We tend to only use quotes when it's strictly necessary, and use single quotes first (for various reasons).

Reply:
Thanks for letting me know. I have fixed this accordingly. Kindly recheck.